kani-compiler/src/codegen_cprover_gotoc/codegen/function.rs

```diff
@@ -58,7 +67,8 @@ impl<'tcx> GotocCtx<'tcx> {
             let loc = self.codegen_span(&ldata.source_info.span);
             let sym =
                 Symbol::variable(name, base_name, t, self.codegen_span(&ldata.source_info.span))
-                    .with_is_hidden(!ldata.is_user_variable());
+                    .with_is_hidden(!ldata.is_user_variable())
+                    .with_is_parameter(idx > 0 && idx <= num_args);
```

Comment:
I don't know why this condition is here? Definitely calls for a comment.

Reply:
yeah... that's a good point. I'll add some comments.

Reply:
What's 0? And the rest are locals I assume?

Reply:
0 is the return variable. There is a comment right below that talks about that.